src/renderers/native/ReactNativeFiber.js

```diff
@@ -372,8 +372,9 @@ const ReactNative = {
     const root = roots.get(containerTag);
     if (root) {
       // TODO: Is it safe to reset this now or should I wait since this unmount could be deferred?
-      roots.delete(containerTag);
-      NativeRenderer.unmountContainer(root);
+      NativeRenderer.updateContainer(null, root, null, () => {
+        roots.delete(containerTag);
```

Comment:
Actually for a deep error I think they would because we catch unmount errors during the commit phase and continue committing the rest of the effect list.